src/main/java/com/urswolfer/gerrit/client/rest/http/config/ServerRestClient.java

```diff
@@ -38,7 +41,8 @@ public ServerRestClient(GerritRestClient gerritRestClient) {
     public String getVersion() throws RestApiException {
```

Comment:
I think that returns the exact version, but your lookup is based on the minor version only.

https://review.gerrithub.io/config/server/version returns: `3.1.2-28-g6d57599017`

Reply:
Oh yes, good catch. I am also missing some tests about it. Going to address it now.

Reply:
It should be fixed now. Can you please re-check?